### PR TITLE
create custom error 500 page

### PR DESCRIPTION
**Pull request by subindevs**
created custom error 500 page. Currently using error coding.png as the page icon. Need to create a new error icon for error 500

### CODE REVIEW

**Review by abhiprojectz — APPROVED** [review]
Working fine

File: dashboard/templates/dashboard/500.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+{% load static %}
+
+{% block navbar %} {% endblock %}
+
+{% block content %}
+
+<div class="error-page-wrapper text-center">
+    <img src="{% static 'error_icons/coding.png' %}" alt="{% trans 'Internal Server Error' %}">
+    <h3> {% trans 'Sorry something has gone wrong on the server!' %}</h3>
```
Comment (pskeshu) [inline dashboard/templates/dashboard/500.html:16]: I was wondering whether there is any benefit in not hardcoding the text in the template.

**Review by pskeshu — APPROVED** [review]
looks good, for the scope of this PR.

**Comment by jr-sawaliram** [comment]
fixes #351

**Pull request merged**